### PR TITLE
SeventhString.Transcribe version 0.9.0.20

### PR DESCRIPTION
**Pull request by chausner**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/50218)

### CODE REVIEW

File: manifests/s/SeventhString/Transcribe/0.9.0.20/SeventhString.Transcribe.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: SeventhString.Transcribe
+PackageVersion: 0.9.0.20
```
Comment (Trenly) [inline manifests/s/SeventhString/Transcribe/0.9.0.20/SeventhString.Transcribe.installer.yaml:5]:
```suggestion
PackageVersion: 9.20
```
This must match control panel; Remember to update the name of the version folder when updating this field

File: manifests/s/SeventhString/Transcribe/0.9.0.20/SeventhString.Transcribe.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: SeventhString.Transcribe
+PackageVersion: 0.9.0.20
```
Comment (Trenly) [inline manifests/s/SeventhString/Transcribe/0.9.0.20/SeventhString.Transcribe.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 9.20
```

File: manifests/s/SeventhString/Transcribe/0.9.0.20/SeventhString.Transcribe.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: SeventhString.Transcribe
+PackageVersion: 0.9.0.20
+Installers:
+- Architecture: x64
+  InstallerType: inno
+  InstallerUrl: https://www.seventhstring.com/xscribe/xscsetup.exe
+  InstallerSha256: 7686678AE0BC4195C1ED08CA971BA1C0518561BFF4E024A48A0A96775A6A8D48
```
Comment (Trenly) [inline manifests/s/SeventhString/Transcribe/0.9.0.20/SeventhString.Transcribe.installer.yaml:10]:
```suggestion
  InstallerSha256: 7686678AE0BC4195C1ED08CA971BA1C0518561BFF4E024A48A0A96775A6A8D48
  ProductCode: com.seventhstring.Transcribe_is1
```

File: manifests/s/SeventhString/Transcribe/0.9.0.20/SeventhString.Transcribe.yaml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: SeventhString.Transcribe
+PackageVersion: 0.9.0.20
```
Comment (Trenly) [inline manifests/s/SeventhString/Transcribe/0.9.0.20/SeventhString.Transcribe.yaml:5]:
```suggestion
PackageVersion: 9.20
```

**Review by Trenly — CHANGES_REQUESTED** [review]

**Review by ImJoakim — APPROVED** [review]

**Pull request merged**